src/test/java/net/sf/jabref/remote/TestConnector.java

```diff
@@ -0,0 +1,27 @@
+package net.sf.jabref.remote;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class TestConnector {
+
+    public static DBType currentConnectionType;
+
+
+    public static Connection getTestConnection(DBType dbType) throws ClassNotFoundException, SQLException {
+        String user = "root";
+        String password = "";
+        String database = "jabref";
+
+        if (dbType == DBType.POSTGRESQL) {
+            user = "postgres";
+        } else if (dbType == DBType.ORACLE) {
+            user = "travis";
+            password = "travis";
+            database = "xe";
+        }
+        currentConnectionType = dbType;
+
+        return DBConnector.getNewConnection(dbType, "localhost", database, user, password);
```

Comment:
How do I make certain that there is a right database listening at `localhost` before I run the tests?